cmd/demo/demo.go

```diff
@@ -25,21 +23,28 @@ var demoCmd = &cobra.Command{
 	Run: runDemo,
 }
 
-var testAPI bool
+var testAPIFlag bool
```

Comment:
I think we should also put these options in the `Config` to have them all bundled. Maybe create another struct `demoConfig` which holds `testAPIEnabled` (rename from `testAPIFlag`), `cfgFile` and `cfgNetFile`.

Reply:
ok, i created a struct for the command line flags.